config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-periodics.yaml.tpl

```diff
@@ -274,4 +274,61 @@ periodics:
     testgrid-tab-name: capi-e2e-conformance-ci-latest-{{ ReplaceAll $.branch "." "-" }}
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
+- name: periodic-cluster-api-e2e-{{ ReplaceAll $.branch "." "-" }}
```

Comment:
Let's change this name so we end up with e.g. "pull-cluster-api-e2e-latestk8s-main" (similar for the periodic)

Reply:
```suggestion
- name: periodic-cluster-api-e2e-latestk8s-{{ ReplaceAll $.branch "." "-" }}
```

Reply:
```suggestion
{{ if eq $.branch "main" }}
- name: periodic-cluster-api-e2e-{{ ReplaceAll $.branch "." "-" }}
```

And further down at the end of the new job
{{ end }}

Reply:
Added, thanks for clarifying that @chrischdi